sections/chapters/introduction/overview.tex

```diff
@@ -1,11 +1,55 @@
-As previously mention our goal is to design and develop a tool that will asssist the user in gathering insightful data from unstructured data.
-In order to achieve this target we decided to implement various methods into our project. Figure 1.1 displays the architecture designed for this specific project.  
+As previously mentioned, our goal is to design and develop a tool that will asssist the user in gathering insightful data from unstructured data.
```

Comment:
I really hate the word "unstructured" I think unorganized works so much better. I guess it's too late to change it everwhere

Reply:
I can do a quick search

Reply:
Done